src/NewCommand.php

```diff
@@ -448,29 +448,70 @@ protected function installJetstream(string $directory, InputInterface $input, Ou
      */
     protected function promptForDatabaseOptions(string $directory, InputInterface $input)
     {
-        $defaultDatabase = 'sqlite';
+        $defaultDatabase = collect(
+            $databaseOptions = $this->databaseOptions()
+        )->keys()->first();
 
         if (! $input->getOption('database') && $input->isInteractive()) {
             $input->setOption('database', select(
                 label: 'Which database will your application use?',
-                options: [
-                    'mysql' => 'MySQL',
-                    'mariadb' => 'MariaDB',
-                    'pgsql' => 'PostgreSQL',
-                    'sqlite' => 'SQLite',
-                    'sqlsrv' => 'SQL Server',
-                ],
-                default: $defaultDatabase
+                options: $databaseOptions,
+                default: $defaultDatabase,
+                validate: fn (string $engine) => ! $this->isDatabaseEngineAvailable($engine)
+                    ? 'You selected an engine with a missing PDO extension. Please select an available option.'
+                    : null
             ));
 
             if ($input->getOption('database') !== $defaultDatabase) {
```

Comment:
If we go with my suggestions, this should be hardcoded to `sqlite` (to match the default in `laravel/laravel`) rather than the computed default, which may not be `sqlite`.

```suggestion
            if ($input->getOption('database') !== 'sqlite') {
```

Reply:
Good thinking